WhatsAppNotification/VirtoCommerce.WhatsAppNotification.Web/Module.cs

```diff
@@ -0,0 +1,60 @@
+using System.IO;
+using System.Reflection;
+using Microsoft.Practices.Unity;
+using VirtoCommerce.Domain.Order.Events;
+using VirtoCommerce.Platform.Core.Bus;
+using VirtoCommerce.Platform.Core.Modularity;
+using VirtoCommerce.Platform.Core.Notifications;
+using VirtoCommerce.WhatsAppNotification.Core.Gateways;
+using VirtoCommerce.WhatsAppNotification.Core.Notifications;
+using VirtoCommerce.WhatsAppNotification.Data.Gateway;
+using VirtoCommerce.WhatsAppNotification.Data.Handlers;
+using VirtoCommerce.WhatsAppNotification.Data.Senders;
+
+namespace VirtoCommerce.WhatsAppNotification.Web
+{
+    public class Module : ModuleBase
+    {
+        private readonly IUnityContainer _container;
+
+        public Module(IUnityContainer container)
+        {
+            _container = container;
+        }
+
+        public override void Initialize()
+        {
+
+            _container.RegisterType<IWhatsAppNotificationSendingGateway, WhatsAppNotificationSendingGateway>();
+            _container.RegisterType<IWhatsAppClient, TwilioWhatsApp>();
+
+            var notificationManager = _container.Resolve<INotificationManager>();
+
+            notificationManager.RegisterNotificationType(() => new OrderWhatsAppNotification(_container.Resolve<IWhatsAppNotificationSendingGateway>())
+            {
+                NotificationTemplate = new NotificationTemplate()
+                {
+                    Body = ReadFile($"{typeof(OrderWhatsAppNotification).Name}_body.htm")
+                }
+            });
+
+            var eventHandlerRegistrar = _container.Resolve<IHandlerRegistrar>();
+
+            eventHandlerRegistrar.RegisterHandler<OrderChangedEvent>(async (message, token) => await _container.Resolve<SendNotificationOrderChangedEventHandler>().Handle(message));
+        }
+
+        private static string ReadFile(string fileName)
```

Comment:
remove this method. Refactor to use .ReadToString() extension from namespace VirtoCommerce.Platform.Core.Common